### ISSUE TITLE
TRANSLATION_GUIDE.md 용어 제안

### ISSUE THREAD

**Issue by uddk6215** [PyTorchKorea/tutorials-kr#819]
## 이슈 내용
용어집의 용어 사용 규칙을 추가합니다:
- Gradient descent : 경사하강법
- Pooling : 풀링

## 추가 정보
Na
